test/public/runs/overview.test.js

```diff
@@ -357,4 +359,49 @@ module.exports = () => {
         expect(String(startMin)).to.equal(minTime);
         expect(String(endMin)).to.equal(minTime);
     });
+
+    it('Should successfully filter runs by their run quality', async () => {
+        await page.goto(`${url}?page=run-overview`, { waitUntil: 'networkidle0' });
+        const filterInputSelectorPrefix = '#runQualityCheckbox';
+        const badFilterSelector = `${filterInputSelectorPrefix}bad`;
+        const testFilterSelector = `${filterInputSelectorPrefix}test`;
+
+        /**
+         * Checks that all the rows of the given table have a valid run quality
+         *
+         * @param {{evaluate: function}[]} rows the list of rows
+         * @param {string[]} authorizedRunQualities  the list of valid run qualities
+         * @return {void}
+         */
+        const checkTableRunQualities = async (rows, authorizedRunQualities) => {
+            for (const row of rows) {
+                expect(await row.evaluate((rowItem) => {
+                    const rowId = rowItem.id;
+                    return document.querySelector(`${rowId}-runQuality-text`).innerText;
+                })).to.be.oneOf(authorizedRunQualities);
+            }
+        };
+
+        // Open filter toggle
+        await pressElement(page, '#openRunFilterToggle');
+        await page.waitForTimeout(1000);
+        takeScreenshot(page);
```

Comment:
I believe there is no need to take a screenshot anymore here (same with the import for `takeScreenshot`)

Reply:
Done